src/diff.c

```diff
@@ -82,6 +82,21 @@ lyd_diff_add(const struct lyd_node *node, enum lyd_diff_op op, const char *orig_
 
     assert(diff);
 
+    /* replace leaf always needs orig-default and orig-value */
```

Comment:
Next time, I don't think that adding asserts is a FEATURE, it's just debugging not even changing anything in the Release build.

Reply:
Okay but these asserts have somewhat extended functionality of documenting (by code) what metadata exactly are expected in which case, that is why using MAINTENANCE or some similar tag did not seems appropriate to me.

Reply:
so then it's DOC, isn't it?

Reply:
Not sure, can DOC be something non-comment?

Reply:
can FEATURE be something that is not compiled? :)

Reply:
Like I said, I do not know what exactly I was supposed to use because none of the tags seem to be suitable.